### ISSUE TITLE
Add Heatmap to UMAP scatterplot

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#126]
The data for the UMAP scatterplot should also support a heatmap. Make the necessary tweaks in https://github.com/hubmapconsortium/portal-containers/ and then update the view config here. (Will be blocked by the lack of a File API.)

**Comment by mccalluc** [comment]
@mruffalo : Can you comment on this? Behind the UMAP, there is presumably higher dimensional data... would it be useful to have a heatmap for that? Or is there a better person to ask?

**Comment by mccalluc** [comment]
@ilan-gold : Do you know if heatmaps would be interesting or useful with this data, or is this just something different?

**Comment by ilan-gold** [comment]
It's on the radar here: https://github.com/hubmapconsortium/portal-containers/issues/30.  The main potential issue is the size - the cell x gene space is much larger than anything we have done previously, on an order of 1000s x 1000s from what I understand.  Also, I'm not sure the data comes through right now - we may have to ask for it, but I could be wrong.

**Comment by mruffalo** [comment]
The expression data (normalized, filtered, etc.) is present in the `h5ad` files you're reading for conversion to Arrow format -- right now you're only pulling out the UMAP coordinates and cluster assignments.

This is stored in the `X` (data matrix) attribute of the `AnnData` object:
```python
In [1]: import anndata                                                                          

In [2]: d = anndata.read_h5ad('cluster_marker_genes.h5ad')                                      

In [3]: d.X                                                                                     
Out[3]: 
array([[ 0.05571311, -0.07759782, -0.0441038 , ..., -0.04950992,
        -0.05211497,  1.8924463 ],
       [-0.4334068 , -0.07759782, -0.0441038 , ..., -0.04950992,
        -0.05211497, -0.31383678],
       [ 0.52023876, -0.07759782, -0.0441038 , ..., -0.04950992,
        -0.05211497,  0.45089713],
       ...,
       [-0.4334068 , -0.07759782, -0.0441038 , ..., -0.04950992,
        -0.05211497, -0.31383678],
       [-0.4334068 , -0.07759782, -0.0441038 , ..., -0.04950992,
        -0.05211497, -0.31383678],
       [-0.4334068 , -0.07759782, -0.0441038 , ..., -0.04950992,
        -0.05211497, -0.31383678]], dtype=float32)
```

**Comment by keller-mark** [comment]
Closing this, duplicate of https://github.com/hubmapconsortium/portal-ui/issues/696
